src/Dfe.PlanTech.Domain/Questionnaire/Models/IHasContentComponent.cs

```diff
@@ -0,0 +1,12 @@
+using Dfe.PlanTech.Domain.Content.Models;
+
+namespace Dfe.PlanTech.Domain.Questionnaire.Models;
+
+public interface IHasContentComponent
```

Comment:
Not a huge fan of the name but I'll let it slide

Reply:
Suggestion? I suck with names

Reply:
I don't like IIsContentComponent - (hate interfaces with a double i)......

Reply:
just leave it for now